projects/plugins/jetpack/extensions/blocks/videopress/resumable-upload/use-uploader.js

```diff
@@ -109,9 +195,8 @@ export const resumableUploader = ( { onUploadUuidRetrieved, onError, onProgress,
 
 				if ( [ 'OPTIONS', 'GET', 'HEAD', 'DELETE', 'PUT', 'PATCH' ].indexOf( method ) >= 0 ) {
 					const url = new URL( req._url );
-					const path = url.pathname;
-					const parts = path.split( '/' );
-					const maybeUploadkey = parts[ parts.length - 1 ];
+					const maybeUploadkey = extractUploadKeyForUrl( url );
+					completionPolling().start( upload.url, onSuccess, onError );
```

Comment:
> When upload is about to complete, start polling for a guid in addition to waiting for a response from the final chunk

During my testing I added some logs when the polling interval was occurring, and it seemed to start polling right away after the upload started. Shouldn't there be some checks if the progress is > 95% or something?